drivers/hmis/app/graphql/mutations/base_mutation.rb

```diff
@@ -6,109 +6,12 @@
 
 module Mutations
   class BaseMutation < GraphQL::Schema::RelayClassicMutation
+    include GraphqlMutationHelper
```

Comment:
I'm getting an error here `uninitialized constant Mutations::BaseMutation::GraphqlMutationHelper include GraphqlMutationHelper ^^^^^^^^^^^^^^^^^^^^^`

Reply:
Oh sorry you might need to restart your rails server, lmk if that doesn't work